### PR TITLE
Add more feeds to DailyBuilds.md

### PR DESCRIPTION
**Pull request by dougbu**
- #9170

### CODE REVIEW

**Comment by dougbu** [comment]
@danroth27 please confirm your previously-failing tests work with this new configuration

**Comment by davidfowl** [comment]
Why do we need this many feeds now?

**Comment by dougbu** [comment]
@davidfowl root cause is the lock contention we and our partners ran into when everything was pushing to a single blob store. (Our partners are considering breaking up the core-setup store as well.)

Good news is this is temporary. Will be moving to entirely new feeds&hellip;

**Review by danroth27 — APPROVED** [review]

File: docs/DailyBuilds.md [hunk 1]
```diff
@@ -14,6 +14,10 @@ If you want to download the latest daily build and use it in a project, then you
       <packageSources>
           <clear />
           <add key="dotnet-core" value="https://dotnetfeed.blob.core.windows.net/dotnet-core/index.json" />
+          <add key="extensions" value="https://dotnetfeed.blob.core.windows.net/aspnet-extensions/index.json" />
+          <add key="entityframeworkcore" value="https://dotnetfeed.blob.core.windows.net/aspnet-entityframeworkcore/index.json" />
+          <add key="aspnetcore-tooling" value="https://dotnetfeed.blob.core.windows.net/aspnet-aspnetcore-tooling/index.json;" />
```
Comment (natemcmaster) [inline docs/DailyBuilds.md]: Does the semicolon at the end cause issue for the NuGet client? (applies to the next line, too)

Comment (dougbu) [reply docs/DailyBuilds.md]: Good catch. I'm not sure about what problems it might cause but I'll clean these up in a moment&hellip;

**Pull request merged**